### PR TITLE
[nrfconnect] Updated samples docs and user guides.

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
#### Problem
After some code changes documentation is out of date.

 #### Summary of Changes
* In samples docs added reference
to the nrfconnect_android_commissioning.md
* Updated nrfconnect_android_commissioning.md to the current
CHIP Tool UI version.

### CODE REVIEW

File: docs/guides/nrfconnect_android_commissioning.md [hunk 1]
```diff
@@ -141,13 +141,12 @@ To form a Thread network, complete the following steps:
 
         $ docker network create --ipv6 --subnet 2001:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
 
-2.  Start the OTBR container using the command below with _rcp-dk_ in the last
-    line replaced with the device node name of the nRF52840 DK that is running
-    the RCP firmware (for example, _/dev/ttyACM0_):
+2.  Start the OTBR container using the device node name of the nRF52840 DK
```
Comment (Damian-Nordic) [inline docs/guides/nrfconnect_android_commissioning.md:144]: I remember last time someone requested the information that the word which should be replaced is in the last line as it wasn't obvious where to look for it.

File: docs/guides/nrfconnect_android_commissioning.md [hunk 2]
```diff
@@ -290,26 +289,23 @@ following steps:
...
 3. Open the CHIPTool application on your smartphone.
-4. Tap the **SCAN QR CODE** button and scan the commissioning QR code.
-5. Tap the **Rendezvous over BLE** button to initiate the commissioning
-   procedure. You will see a few pop-up messages appear as the commissioning
+4. Tap the **PROVISION CHIP DEVICE WITH THREAD** button and scan the commissioning QR code.
+5. You will see a few pop-up messages appear as the commissioning
    progresses. Eventually, the network settings screen appears.
-6. In the new screen, open the **THREAD** tab.
-7. Tap the **Save Network** button to send a Thread provisioning message to the
+6. In the new screen tap the **Save Network** button to send a Thread provisioning message to the
```
Comment (Damian-Nordic) [inline docs/guides/nrfconnect_android_commissioning.md:295]: nit: Add a comma after "In the new screen".

File: docs/guides/nrfconnect_android_commissioning.md [hunk 3]
```diff
@@ -290,26 +289,23 @@ following steps:
...
    accessory device.
 
 You will see the "Network provisioning completed" message when the accessory
 device successfully joins the Thread network.
 
 ## Sending CHIP commands
 
-Once the device is commissioned, check the connection with the following
-commands:
+Once the device is commissioned, you should be able to see new window containing
```
Comment (Damian-Nordic) [inline docs/guides/nrfconnect_android_commissioning.md:303]: I think @greg-fer usually prefers "something happens" over "something should happen" in such cases. How about: "Once the device is commissioned, a new screen with a text box and **ON**, **TOGGLE**, **OFF** buttons appears"? Not sure we need to mention the slider if we don't use it below.

File: docs/guides/nrfconnect_android_commissioning.md [hunk 4]
```diff
@@ -290,26 +289,23 @@ following steps:
...
+text box, three buttons  - ON, TOGGLE, OFF and a slider. Check the connection with the following
+steps:
 
-1. Go back to the main application screen.
-2. Tap the **LIGHT ON/OFF CLUSTER** button.
-3. Verify that the text box on the next screen contains the IPv6 address of the
+1. Verify that the text box on the next screen contains the IPv6 address of the
```
Comment (Damian-Nordic) [inline docs/guides/nrfconnect_android_commissioning.md:307]: remove "next"?

File: docs/guides/nrfconnect_android_commissioning.md [hunk 5]
```diff
@@ -290,26 +289,23 @@ following steps:
...
    accessory device.
-4. Tap either the **ON** or the **OFF** button to lock or unlock the door,
-   respectively. The **LED 2** on the device turns on or off based on the
-   changes of the lock state.
+2. Tap either the **ON** or the **OFF** button to lock or unlock the door,
+   respectively. The **TOGGLE** button changes lock state to the opposite.
```
Comment (Damian-Nordic) [inline docs/guides/nrfconnect_android_commissioning.md:310]: changes **the** lock state

File: docs/guides/nrfconnect_android_commissioning.md (hunk 1 above)
Comment (kkasperczyk-no) [reply docs/guides/nrfconnect_android_commissioning.md:144]: @LuDuda what is your opinion? We were talking about removing `_rcp-dk_`, as Łukasz reported that people had a problem to find it anyway, so the blog post was already modified and I only wanted to align this instruction to blogpost.

Comment (Damian-Nordic) [reply docs/guides/nrfconnect_android_commissioning.md:144]: I'm fine with removing _rcp-dk_ :) I just think we may leave information that `/dev/ttyACM0` in the last line should be replaced.

Comment (kkasperczyk-no) [reply docs/guides/nrfconnect_android_commissioning.md:144]: Oh ok, I didn't get it. I will add it.

File: docs/guides/nrfconnect_android_commissioning.md (hunk 3 above)
Comment (kkasperczyk-no) [reply docs/guides/nrfconnect_android_commissioning.md]: Partially done, I rephrased it, but I would not remove slider, as it is a part of appearing UI, even if we are not using it.

**Review by greg-fer — APPROVED** [review]

**Pull request closed without merging**